frame/dvm/src/lib.rs

```diff
@@ -81,6 +81,52 @@ impl Default for EthereumStorageSchema {
 	}
 }
 
+/// The ethereum transaction include recovered source account
+pub struct EthereumTransaction {
```

Comment:
Two suggestions here:
1. We can move this struct at the end of  https://github.com/darwinia-network/darwinia-common/blob/master/primitives/rpc/src/transaction.rs.
2. `EthereumTransaction` rename to `InternalTransaction` would be better.

Reply:
I found that `EthereumTransaction` is used temporally in memory in this file, and also used by normal transaction.

I'm changing it to DVMTransaction.

Reply:
Yes, it's used to convert the normal transaction to the format `execute` fn used.